benchmarks/src/main/java/org/questdb/CsvLexerBenchmark.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ *     ___                  _   ____  ____
+ *    / _ \ _   _  ___  ___| |_|  _ \| __ )
+ *   | | | | | | |/ _ \/ __| __| | | |  _ \
+ *   | |_| | |_| |  __/\__ \ |_| |_| | |_) |
+ *    \__\_\\__,_|\___||___/\__|____/|____/
+ *
+ *  Copyright (c) 2014-2019 Appsicle
+ *  Copyright (c) 2019-2022 QuestDB
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ ******************************************************************************/
+
+package org.questdb;
+
+import io.questdb.cutlass.text.CsvTextLexer;
+import io.questdb.cutlass.text.DefaultTextConfiguration;
+import io.questdb.std.Files;
+import io.questdb.std.MemoryTag;
+import io.questdb.std.Unsafe;
+import io.questdb.std.str.Path;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class CsvLexerBenchmark {
+    public static void main(String[] args) {
+        long BUF_SIZE = 64 * 1024;
+        long buf = Unsafe.malloc(BUF_SIZE, MemoryTag.NATIVE_DEFAULT);
+        long fd = Files.openRO(new Path().of("C:\\Users\\Vlad\\dev\\csv-parsers-comparison\\worldcitiespop.txt").$());
```

Comment:
nit: would be nice to avoid concrete dev environment dependencies and, maybe, add a comment with a link to download this file (unless we want to include it into our repo).

Reply:
It is a big file, 144mb unpacked. I will add comments.

These days the issue is that there isn’t one url to get this file from. I would recommend googling it?

Reply:
done in https://github.com/questdb/questdb/pull/2576/commits/bb7502c6c4bf1ddee8b56d7c81785d59ca672d7c